index.html

```diff
@@ -26,16 +26,131 @@ <h1 class="primaryHeading">
+            <ul class="language">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javacript</li>
+            </ul>
+            <a href="#" class="seeProject">See Project</a>
+        </div>
+
+
+    <div class="card">
+        <div class="image3 img"></div>
+        <h1 class="tonic">Multi-Post Stories</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+            <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+
+
+    <div class="card">
+        <div class="image4 img"></div>
+        <h1 class="tonic">Tonic</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+        <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+
+
+    <div class="card">
+        <div class="image5 img"></div>
+        <h1 class="tonic">Multi-Post Stories</h1>
+
+        <div class="divList">
+            <p class="canopy">canopy</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">Back End Dev</p>
+            <img class="listIcon" src="Counter.png"><p class="workList">2022</p>
+        </div>
+
+        <p class="workText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="language">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javascript">javacript</li>
+        </ul>
+        <a href="#" class="seeProject">See Project</a>
+    </div>
+</section>  
+
+<section class="about">
+    <h1 class="aboutHeading">
+        About Myself
+    </h1>
+    <p class="mainContent">
+        I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+    </p>
+
+    <a href="#" class="connect connectAbout">Let's Connect</a>
```

Comment:
- [ ] Kindly use hx as this should not link the user to anywhere. Only the links below are expected to link the user.